app/models/merchant.rb

```diff
@@ -4,27 +4,40 @@ class Merchant < ApplicationRecord
   def favorite_customers
     Customer.joins(invoices: [:transactions, [invoice_items: [item: [:merchant]]]])
       .where(transactions: { result: 'success' })
-      .where(merchants: { id: id })  #second id is the id of the merchant that's getting called out
+      .where(merchants: { id: id })
       .group(:id)
       .order(Arel.sql('COUNT(transactions.id) DESC'))
       .limit(5)
+      .select(
+        "customers.first_name as first_name,
+        customers.last_name as last_name,
```

Comment:
Would it be beneficial to create a helper method to dry up adding the first and last name strings? I feel like we could use that in a few places throughout the project.

Reply:
possibly! I think we could also add them together in the SQL if we wanted to, like `"customers.first_name + ' ' + customers.last_name as full_name"` or something